packages/eslint-plugin/tests/rules/component-selector.test.ts

```diff
@@ -341,5 +373,33 @@ ruleTester.run(RULE_NAME, rule, {
       ],
       data: { type: 'attribute' },
     }),
+    convertAnnotatedSourceToFailureCase({
+      description: `it should fail if a ShadowDom selector is not kebab-case`,
+      annotatedSource: `
+      @Component({
+        encapsulation: ViewEncapsulation.ShadowDom,
+        selector: 'appFooBar'
+                  ~~~~~~~~~~~
+      })
+      class Test {}
+      `,
+      messageId: messageIdStyleFailure,
+      options: [{ type: 'element', prefix: ['app'], style: 'camelCase' }],
+      data: { style: 'kebab-case' },
+    }),
+    convertAnnotatedSourceToFailureCase({
+      description: `it should fail if a ShadowDom selector is contains hyphen`,
```

Comment:
```suggestion
      // https://github.com/angular-eslint/angular-eslint/issues/534
      description: `it should fail if a ShadowDom-encapsulated component's selector doesn't contain hyphen`,
```

Reply:
Hey @GavinWu1991, did you have a chance to see the suggestion for this description and the one above?

Reply:
Sorry, I guess I missed the suggestion for the description.